<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-M4N4JHAP" ignored="0" name="Subtraction" tests="7" failures="7" timestamp="2021-03-15T10:01:10 GMT" time="50.509" errors="0">
  <testcase name="SubtractAscendRangeAndDescendRange" time="6.951" classname="Acacemy.SubtractionTest">
    <failure type="java.lang.AssertionError" message="expected [-55545] but found [690235.0000000001]">
      <![CDATA[java.lang.AssertionError: expected [-55545] but found [690235.0000000001]
at Acacemy.SubtractionTest.SubtractAscendRangeAndDescendRange(SubtractionTest.java:33)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- SubtractAscendRangeAndDescendRange -->
  <testcase name="SubtractBracketsFiveAndTwo" time="5.551" classname="Acacemy.SubtractionTest">
    <failure type="java.lang.AssertionError" message="expected [-7] but found [7.0000000001]">
      <![CDATA[java.lang.AssertionError: expected [-7] but found [7.0000000001]
at Acacemy.SubtractionTest.SubtractBracketsFiveAndTwo(SubtractionTest.java:67)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- SubtractBracketsFiveAndTwo -->
  <testcase name="SubtractDecimalTwoPointTwoMinusOne" time="5.323" classname="Acacemy.SubtractionTest">
    <failure type="java.lang.AssertionError" message="expected [3.2] but found [3.2000000001]">
      <![CDATA[java.lang.AssertionError: expected [3.2] but found [3.2000000001]
at Acacemy.SubtractionTest.SubtractDecimalTwoPointTwoMinusOne(SubtractionTest.java:73)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- SubtractDecimalTwoPointTwoMinusOne -->
  <testcase name="SubtractMorethanTwoNumbers" time="5.379" classname="Acacemy.SubtractionTest">
    <failure type="java.lang.AssertionError" message="expected [3] but found [15.0000000001]">
      <![CDATA[java.lang.AssertionError: expected [3] but found [15.0000000001]
at Acacemy.SubtractionTest.SubtractMorethanTwoNumbers(SubtractionTest.java:80)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- SubtractMorethanTwoNumbers -->
  <testcase name="SubtractOneAndZero" time="5.207" classname="Acacemy.SubtractionTest">
    <failure type="java.lang.AssertionError" message="expected [-6] but found [78.0000000001]">
      <![CDATA[java.lang.AssertionError: expected [-6] but found [78.0000000001]
at Acacemy.SubtractionTest.SubtractOneAndZero(SubtractionTest.java:61)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- SubtractOneAndZero -->
  <testcase name="SubtractZeroAndOne" time="5.204" classname="Acacemy.SubtractionTest">
    <failure type="java.lang.AssertionError" message="expected [2] but found [2.0000000001]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [2.0000000001]
at Acacemy.SubtractionTest.SubtractZeroAndOne(SubtractionTest.java:51)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- SubtractZeroAndOne -->
  <testcase name="SubtractZeroAndZero" time="5.201" classname="Acacemy.SubtractionTest">
    <failure type="java.lang.AssertionError" message="expected [0] but found [2.0000000001]">
      <![CDATA[java.lang.AssertionError: expected [0] but found [2.0000000001]
at Acacemy.SubtractionTest.SubtractZeroAndZero(SubtractionTest.java:40)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- SubtractZeroAndZero -->
</testsuite> <!-- Subtraction -->
